### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by troutowicz**
Add a note about react-stamp and it replacing this library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 Create React components in a way analogous to `React.createClass`, but powered by a [subset](docs/api.md#api-differences) of the [stampit](https://github.com/stampit-org/stampit) API.
 
+**This library is no longer maintained and is being replaced by the stamp spec compliant [react-stamp](https://github.com/troutowicz/react-stamp).**
```
Comment (ericelliott) [inline README.md:9]: Maybe we should wait for `react-stamp` to be stable before adding this notice?

Comment (troutowicz) [inline README.md:9]: Either way is fine with me, but is there anything left to maintain here?

Comment (ericelliott) [inline README.md:9]: Probably not, but do we want to send the message, "This repo is obsolete, and the new one is not ready and unstable"?

Comment (troutowicz) [inline README.md:9]: Fair.

**Pull request closed without merging**